arenadata/scripts/run_resgroup_test.bash

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+project="resgroup"
+
+#install gpdb and setup gpadmin user
+bash arenadata/scripts/init_containers.sh $project
+
+for service in 'cdw' 'sdw1'
+do
+  #grant access rights to group controllers
+  docker-compose -p $project -f arenadata/docker-compose.yaml exec $service bash -c "
+    chmod -R 777 /sys/fs/cgroup/{memory,cpu,cpuset} &&
+    mkdir /sys/fs/cgroup/{memory,cpu,cpuset}/gpdb &&
+    chmod -R 777 /sys/fs/cgroup/{memory,cpu,cpuset}/gpdb &&
+    chown -R gpadmin:gpadmin /sys/fs/cgroup/{memory,cpu,cpuset}/gpdb"
+done
+
+#create cluster
+docker-compose -p $project -f arenadata/docker-compose.yaml exec cdw \
+ bash -c "source gpdb_src/concourse/scripts/common.bash && HOSTS_LIST='sdw1' make_cluster"
+
+#run tests
+docker-compose -p $project -f arenadata/docker-compose.yaml exec -Tu gpadmin cdw bash -ex <<EOF
+        source /usr/local/greenplum-db-devel/greenplum_path.sh
+        source gpdb_src/gpAux/gpdemo/gpdemo-env.sh
+        export LDFLAGS="-L\${GPHOME}/lib"
+        export CPPFLAGS="-I\${GPHOME}/include"
+        export USER=gpadmin
+
+        cd /home/gpadmin/gpdb_src
+        ./configure --prefix=/usr/local/greenplum-db-devel \
+            --without-zlib --without-rt --without-libcurl \
+            --without-libedit-preferred --without-docdir --without-readline \
+            --disable-gpcloud --disable-gpfdist --disable-orca \
+            ${CONFIGURE_FLAGS}
+
+        make -C /home/gpadmin/gpdb_src/src/test/regress
+        ssh sdw1 mkdir -p /home/gpadmin/gpdb_src/src/test/regress </dev/null
```

Comment:
is it possible to remove "/dev/null"?

Reply:
`</dev/null` is needed to detach `stdin` from the process and pass `EOF` to it when trying to read from it. without this, for some reason, the script does not execute after the ssh command.